src/libsyntax/ext/mtwt.rs

```diff
@@ -35,7 +35,7 @@ use std::collections::HashMap;
 pub struct SCTable {
     table: RefCell<Vec<SyntaxContext_>>,
     mark_memo: RefCell<HashMap<(SyntaxContext,Mrk),SyntaxContext>>,
-    rename_memo: RefCell<HashMap<(SyntaxContext,Name,SyntaxContext,Name),SyntaxContext>>,
+    rename_memo: RefCell<HashMap<(SyntaxContext,(Name,SyntaxContext),Name),SyntaxContext>>,
```

Comment:
Just for clarity. It's actually an `Ident` (and it was literally `Ident` before https://github.com/rust-lang/rust/pull/28642), but it should be hashed and compared as pair and not as an Ident.